src/Files.App/Data/Commands/Manager/CommandManager.cs

```diff
@@ -199,149 +199,149 @@ public CommandManager()
-			[CommandCodes.GroupByDateModifiedYear] = new GroupByDateModifiedYearAction(),
-			[CommandCodes.GroupByDateModifiedMonth] = new GroupByDateModifiedMonthAction(),
-			[CommandCodes.GroupByDateCreatedYear] = new GroupByDateCreatedYearAction(),
-			[CommandCodes.GroupByDateCreatedMonth] = new GroupByDateCreatedMonthAction(),
-			[CommandCodes.GroupByDateDeletedYear] = new GroupByDateDeletedYearAction(),
-			[CommandCodes.GroupByDateDeletedMonth] = new GroupByDateDeletedMonthAction(),
-			[CommandCodes.GroupAscending] = new GroupAscendingAction(),
-			[CommandCodes.GroupDescending] = new GroupDescendingAction(),
-			[CommandCodes.ToggleGroupDirection] = new ToggleGroupDirectionAction(),
-			[CommandCodes.GroupByYear] = new GroupByYearAction(),
-			[CommandCodes.GroupByMonth] = new GroupByMonthAction(),
-			[CommandCodes.ToggleGroupByDateUnit] = new ToggleGroupByDateUnitAction(),
-			[CommandCodes.NewTab] = new NewTabAction(),
-			[CommandCodes.FormatDrive] = new FormatDriveAction(),
-			[CommandCodes.NavigateBack] = new NavigateBackAction(),
-			[CommandCodes.NavigateForward] = new NavigateForwardAction(),
-			[CommandCodes.NavigateUp] = new NavigateUpAction(),
-			[CommandCodes.DuplicateCurrentTab] = new DuplicateCurrentTabAction(),
-			[CommandCodes.DuplicateSelectedTab] = new DuplicateSelectedTabAction(),
-			[CommandCodes.CloseTabsToTheLeftCurrent] = new CloseTabsToTheLeftCurrentAction(),
-			[CommandCodes.CloseTabsToTheLeftSelected] = new CloseTabsToTheLeftSelectedAction(),
-			[CommandCodes.CloseTabsToTheRightCurrent] = new CloseTabsToTheRightCurrentAction(),
-			[CommandCodes.CloseTabsToTheRightSelected] = new CloseTabsToTheRightSelectedAction(),
-			[CommandCodes.CloseOtherTabsCurrent] = new CloseOtherTabsCurrentAction(),
-			[CommandCodes.CloseOtherTabsSelected] = new CloseOtherTabsSelectedAction(),
-			[CommandCodes.OpenDirectoryInNewPane] = new OpenDirectoryInNewPaneAction(),
-			[CommandCodes.OpenDirectoryInNewTab] = new OpenDirectoryInNewTabAction(),
-			[CommandCodes.OpenInNewWindowItem] = new OpenInNewWindowItemAction(),
-			[CommandCodes.ReopenClosedTab] = new ReopenClosedTabAction(),
-			[CommandCodes.PreviousTab] = new PreviousTabAction(),
-			[CommandCodes.NextTab] = new NextTabAction(),
-			[CommandCodes.CloseSelectedTab] = new CloseSelectedTabAction(),
-			[CommandCodes.OpenNewPane] = new OpenNewPaneAction(),
-			[CommandCodes.ClosePane] = new ClosePaneAction(),
-			[CommandCodes.OpenFileLocation] = new OpenFileLocationAction(),
-			[CommandCodes.PlayAll] = new PlayAllAction(),
-			[CommandCodes.GitFetch] = new GitFetchAction(),
-			[CommandCodes.GitInit] = new GitInitAction(),
-			[CommandCodes.GitPull] = new GitPullAction(),
-			[CommandCodes.GitPush] = new GitPushAction(),
-			[CommandCodes.GitSync] = new GitSyncAction(),
-			[CommandCodes.OpenAllTaggedItems] = new OpenAllTaggedActions(),
+			[CommandCodes.OpenHelp] = new DebouncedActionDecorator(new OpenHelpAction(), TimeSpan.FromMilliseconds(800)),
```

Comment:
Seeing how often the `TimeSpan.FromMilliseconds(800)` is repeated, why not directly put it inside the constructor of `DebouncedActionDecorator` as a default value, and create a second constructor that'd take a timespan as second argument? 
It'd reduce the repetition, simplify the change of the default value and leaves the room for case to case customization.